src/main/java/in/projecteka/gateway/clients/model/Session.java

```diff
@@ -0,0 +1,31 @@
+package in.projecteka.gateway.clients.model;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
You should create separate PR to use only @Value to make classes immutable.

Reply:
yes i'll do that and create separate PR for the same